documentation/book/proc-configuring-maintenance-time-windows.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// assembly-maintenance-time-windows.adoc
+
+[id='proc-configuring-maintenance-time-windows-{context}']
+= Configuring maintenance time windows
+
```

Comment:
_Procedures must include an introduction._

You can configure a maintenance time window for rolling updates to supported processes.